src/test/java/DatabaseTest.java

```diff
@@ -0,0 +1,79 @@
+import database.Database;
+import database.DatabaseGateway;
+import fridge.Fridge;
+import fridge.Ingredient;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+
+public class DatabaseTest {
+
+    private final String username = "Eric";
+    private DatabaseGateway database;
+    private Fridge fridge;
+
+    /**
+     * Setup method before each test
+     */
+    @BeforeEach
+    public void setUp() {
+        fridge = new Fridge();
+        Ingredient t = new Ingredient();
```

Comment:
Probably more a question for @Shaffaan. Is this the implementation that's going to be used for Ingredient. Would it be better to have the name passed in as argument?

Reply:
Yes I think Shaffaan has fixed this.